Users/Glob_Patterns.rst

```diff
@@ -174,14 +209,10 @@ brackets. Parentheses that have no match are ignored as well as
 
```

Comment:
**grave accent**  `` ` `` is different from Apostrophe `` ' ``. We use grave accent to highlight text not apostrophe. And under **grave accent** `*` is not a special character. Therefore, no need of `\` .